.werft/vm/vm.ts

```diff
@@ -190,20 +112,19 @@ export function waitForVMReadiness(options: { name: string; timeoutSeconds: numb
  * Copies the k3s kubeconfig out of the VM and places it at `path`
  * If it doesn't manage to do so before the timeout it will throw an Error
  */
-export function copyk3sKubeconfig(options: { name: string; timeoutMS: number; slice: string }) {
+export function copyk3sKubeconfigShell(options: { name: string; timeoutMS: number; slice: string }) {
     const werft = getGlobalWerftInstance();
     const startTime = Date.now();
     while (true) {
         const status = exec(
-            `ssh -i /workspace/.ssh/id_rsa_harvester_vm ubuntu@127.0.0.1 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no 'sudo cat /etc/rancher/k3s/k3s.yaml' > ${PREVIEW_K3S_KUBECONFIG_PATH}`,
-            { silent: true, dontCheckRc: true, slice: options.slice },
+            `VM_NAME=${options.name} \
+                      K3S_KUBECONFIG_PATH=${PREVIEW_K3S_KUBECONFIG_PATH} \
+                      GCLOUD_SERVICE_ACCOUNT_PATH=${GCLOUD_SERVICE_ACCOUNT_PATH} \
+                      ./dev/preview/install-k3s-kubeconfig.sh`,
```

Comment:
We use this script to get the context in a workspace - so I've removed the TS implementation and chose to use only the other one. I'll eventually refactor it.